lib/libutils/isoc/newlib/strrchr.c

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
this may deserve a copyright notice. It is not is the original file but should appear in the op-tee source file tree. See other newlib **.c** and **.h** files as example.

Reply:
ack